core/src/test/scala/org/apache/spark/SparkFunSuite.scala

```diff
@@ -107,6 +108,15 @@ abstract class SparkFunSuite
     getTestResourceFile(file).getCanonicalPath
   }
 
+  protected final def copyAndGetResourceFile(fileName: String, suffix: String): File = {
+    val url = Thread.currentThread().getContextClassLoader.getResource(fileName)
+    // Copy to avoid URISyntaxException during accessing the resources in `sql/core`
```

Comment:
SparkFunSuite is in core, let's make the comment more general, like `accessing the resources from the dependent module.`